app/script.js

```diff
@@ -285,12 +293,80 @@ let quote = {
     }
 }
 
+let focus = {
+    taskChecked: localStorage.getItem("main-focus-check"),
+    task: localStorage.getItem("main-focus"),
+    drawFocus: function () {
+        if (localStorage.getItem("username") !== null) {
+            if (focus.task == null || focus.task =='') {
+                $("#main-focus").html(`
+                <form id="main-focus-form">
+                    <label>What is your main focus (for) today?</label>
```

Comment:
Wich random questions I meant the placeholder: ´placeholder="eg: cooking"´ in the line below